src/Sample.TimeApi/appsettings.json

```diff
@@ -0,0 +1,30 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    },
+    "ApplicationInsights": {
+      "LogLevel": {
+        "Default": "Warning",
+        "Sample": "Information",
+        "Sample.TimeApi.Controllers.TimeController": "Information"
+      }
+    }
+  },
+  "AllowedHosts": "*",
+  "SqlConnectionString": "server=localhost;user id=sa;password=Pass@Word1;",
+  "SampleApp": {
+    "UseApplicationInsights": false,
+    "UseOpenTelemetry": true,
+    "ApplicationInsightsInstrumentationKey": "",
```

Comment:
why it's not configured in `ApplicationInsights` section?

Reply:
Do you mean to move the instrumentation key inside the Logging:ApplicationInsights?